### DISCUSSION TITLE
Italian plugin language issues

### DISCUSSION THREAD

**Discussion by bovirus** [aclap-dev/video-downloadhelper#1098] [Language & Translation]
Hi.

I found some errros in Italian translation for Videohelper plugin.

How to collaborate to fix these issues?

Thanks.

**Comment by jcfrog** [comment]
hi, did you read this? https://github.com/aclap-dev/video-downloadhelper/discussions/110

**Comment by bovirus** [comment]
@jcfrog 

I'm working on this

https://github.com/aclap-dev/vdhlocales

Is it ok?

**Comment by jcfrog** [comment]
You just need to share your updates in a txt file like this user did: https://github.com/aclap-dev/video-downloadhelper/discussions/1101

We will integrate it in vdhlocales.

**Comment by bovirus** [comment]
@jcfrog

The file reported (NL) it's a language change/integration.

The file in github repository is full language file.

I found some mismatches about Italian language file in github repository and file used by plugin v. 9.0.2.12

**Comment by jcfrog** [comment]
You can choose to create the txt file with the extension, or a pull request on the repository.
Important, you will have to test with v9.1 Firefox.

**Comment by bovirus** [comment]
@jcfrog 

About the repository the language files hosted there are only for Firefox addon?

**Reply by paulrouget** [reply]
No, for all browsers.

**Comment by bovirus** [comment]
@jcfrog

I just posted the PR for Italian language.

I made a big job

- aligned english/italian string order 
- added new string missing in Italian language
- revise all Italian strings
- normalize/align translation of same terms.

**Comment by alemodaf** [comment]
[it.txt](https://github.com/user-attachments/files/17486773/it.txt)

Hi, just some adjustment

**Comment by bovirus** [comment]
@alemodaf 

Thanks for the update.
When you have new strings please provide also english version.
I have a doubt about this string

v9_user_message_no_incognito_body = '''Video DownloadHelper non è abilitato nelle finestre private/in incognito. È necessario attivare questa opzione manualmente (non è necessario).'''

In the same string you say "È necessario" (you should do it) and "(non è necessario)" (you should'nt do it).

**Comment by alemodaf** [comment]
Hi, you're right is an understanding. where is (non è necessario) is better
to write (non è richiesto).

You thin i' send again the entire file?

saluti,
Alessandro Modaffari


Il giorno mer 23 ott 2024 alle ore 10:20 bovirus ***@***.***>
ha scritto:

**Comment by bovirus** [comment]
@alemodaf 

The two terms explain an opposite concept

- È necessario = you must do it
- non è richiesto = you couldn't do it

are opposite actions/meanings (one is mandatory - other is an option).
You can change strings in this way

- Puoi = you can do it
- non è obbligatorio = you can also not enable

in both situations are options of an action.
